js/run.js

```diff
@@ -97,7 +97,8 @@ var backends = {
     contentType: "text/plain; charset=UTF-8",
     requestTransform: function(data) {
         return JSON.stringify({
-            source: data.code
+            source: data.code,
+            compiler: location.href.indexOf("prerelease") >= 0 ? "dmd-nightly" : "dmd"
```

Comment:
It's just luck we don't have any symbols in Phobos or Druntime that have "prerelease" in the name.

Can't this at least be `indexOf("-prerelease/")` ?